package.json

```diff
@@ -44,7 +44,6 @@
     "@dnd-kit/utilities": "^3.2.2",
     "@edx/brand": "npm:@openedx/brand-openedx@^1.2.3",
     "@edx/browserslist-config": "1.2.0",
-    "@edx/frontend-component-footer": "^14.3.0",
```

Comment:
I'd like to keep this dependency in here. `frontend-slot-footer` has [an unversioned peer dependency on `frontend-component-footer`](https://github.com/openedx/frontend-slot-footer/blob/b68afc7512bc25ab12553a1427557a6e1f168bf8/package.json#L38), so keeping a versioned dependency in here is helpful.